docs/modules/operation/pages/meta-data.adoc

```diff
@@ -9,7 +9,7 @@ The metadata is a simple triad of strings containing a context, a key and the as
 Each node, each interface and each service can have an arbitrary number of metadata elements assigned to it.
 The only restriction is that the tuple of context and key must be unique in the element with which it is associated.
 
-The association of metadata with nodes, interfaces, and services happens during <<provisioning/detectors/detectors.adoc#ga-provisioning-meta-data, provisioning>>.
+The association of metadata with nodes, interfaces, and services happens during <<reference:configuration/provisioning/introduction.adoc#ref-provisioning-meta-data, provisioning>>.
```

Comment:
Linking on the word "provisioning" leads the user to think the link is discussing the provisioning process, not specifically detectors.
```suggestion
The association of metadata with nodes, interfaces, and services happens during provisioning with the use of <<reference:configuration/provisioning/introduction.adoc#ref-provisioning-meta-data, detectors>>.
```